conda/cli/main_list.py

```diff
@@ -115,6 +115,14 @@ def configure_parser(sub_parsers: _SubParsersAction, **kwargs) -> ArgumentParser
         dest="pip",
         help="Do not include pip-only installed packages.",
     )
+    p.add_argument(
+        "--auth",
+        action="store_true",
+        default=False,
+        dest="auth",
```

Comment:
I think that's the correct incantation for the flip of the argument logic suggested below:

```suggestion
        action="store_false",
        default=True,
        dest="remove_auth",
```